BaseApi.js

```diff
@@ -4,7 +4,7 @@ import * as Device from 'expo-device'
 export const baseUrl = (() => {
   /*global __DEV__*/
   if (__DEV__) {
-    if (Constants.manifest.extra.localAPI) {
+    if (Constants?.expoConfig?.extra?.localAPI) {
```

Comment:
the call below in line 27 also needs fixing :)

Reply:
oops..